DomainContent/LoadTest/Trivia/trivia.js

```diff
@@ -0,0 +1,384 @@
+        icon: TABLET_BUTTON_IMAGE,
+        activeIcon: TABLET_BUTTON_PRESSED
+    });
+    var open = false;
+    var injector;
+    var questionText;
+    var choiceTexts = [];
+    var answerText;
+    var triviaData;
+    var request = Script.require('./modules/request.js').request;
+    var type = null;
+    var category = null;
+    var difficulty = null;
+    var currentChoices = [];
+    var podiumQuestions = [];
+    var podiumReveals = [];
+
+    var htmlEnDeCode = (function() {
+        var charToEntityRegex,
+            entityToCharRegex,
+            charToEntity,
+            entityToChar;
+    
+        function resetCharacterEntities() {
+            charToEntity = {};
+            entityToChar = {};
+            // add the default set
+            addCharacterEntities({
+                '&amp;': '&',
+                '&gt;': '>',
+                '&lt;': '<',
+                '&quot;': '"',
+                '&#39;': "'",
+                '&lsquo;': '"',
+                '&rsquo;': '"',
+                '&ldquo;': '"',
+                '&rdquo;': '"',
+                '&eacute;': 'e',
+                '&prime;': '\'',
+                '&Prime': '"'
+            });
+        }
+    
+        function addCharacterEntities(newEntities) {
+            var charKeys = [],
+                entityKeys = [],
+                key, echar;
+            for (key in newEntities) {
+                echar = newEntities[key];
+                entityToChar[key] = echar;
+                charToEntity[echar] = key;
+                charKeys.push(echar);
+                entityKeys.push(key);
+            }
+            charToEntityRegex = new RegExp('(' + charKeys.join('|') + ')', 'g');
+            entityToCharRegex = new RegExp('(' + entityKeys.join('|') + '|&#[0-9]{1,5};' + ')', 'g');
+        }
+    
+        function htmlEncode(value) {
+            var htmlEncodeReplaceFn = function(match, capture) {
+                return charToEntity[capture];
+            };
+    
+            return (!value) ? value : String(value).replace(charToEntityRegex, htmlEncodeReplaceFn);
+        }
+    
+        function htmlDecode(value) {
+            var htmlDecodeReplaceFn = function(match, capture) {
+                return (capture in entityToChar) ? entityToChar[capture] : String.fromCharCode(parseInt(capture.substr(2), 10));
+            };
+    
+            return (!value) ? value : String(value).replace(entityToCharRegex, htmlDecodeReplaceFn);
+        }
+    
+        resetCharacterEntities();
+    
+        return {
+            htmlEncode: htmlEncode,
+            htmlDecode: htmlDecode
+        };
+    })();
+
+    function begin() {
+        playSound(GAME_BEGIN_SOUND);
+        findTargets();
+        clearGame();
+    }
+  
+    function findTargets() {
+        Entities.findEntities(MyAvatar.position, SEARCH_RADIUS).forEach(function(element) {
+            var name = Entities.getEntityProperties(element, ['name']).name;
```

Comment:
I would suggest checking that name.indexOf("Trivia") here is true before iterating through the switch statement. That way, you don't have to evaluate all of these cases for every other element that isn't a trivia one.